windows/rngallery/Package.appxmanifest

```diff
@@ -13,7 +13,7 @@
 
  <Identity
     Name="48071f82-8a63-40b3-a728-2229057a8f83"
-    Publisher="CN=kmelmon"
+    Publisher="CN=chiaramooney"

```

Comment:
probably need this (and the cert to be CN=Microsoft or Microsoft.ReactNative or something like that

Reply:
Got it! Will be able to upgrade to a Microsoft Cert/Publisher once the app is in the store.